### ISSUE TITLE
Build via GitHub Actions

### ISSUE THREAD

**Issue by andreasgrafen** [FirefoxCSS-Store/FirefoxCSS-Store.github.io#23]
I thew together a really simple Actions workflow to [build the project](https://github.com/FirefoxCSS-Store/FirefoxCSS-Store.github.io/actions?query=workflow%3ABuild) from source on demand. This means that for now this has to be triggered manually.

<img width="346" alt="'Run workflow' button to build the project" src="https://user-images.githubusercontent.com/35840154/106751845-adc11c00-6621-11eb-884a-69b5af7f6bb1.png">
Just hit the **Run Workflow** button and let it do the magic. It will automatically get the npm packages, run the `build` step and then commit everything back into the branch. c:

**Comment by Neikon** [comment]
if we add this . it could be auto
![imagen](https://user-images.githubusercontent.com/1765319/106757106-4363a800-6630-11eb-988c-925436a5e145.png)

**Comment by andreasgrafen** [comment]
While true I did step away from that for now because I don't know how all the Actions stuff actually works.
The [feature page](https://github.com/features/actions) claims that everyone (the free tier) has 2.000 minutes of runtime included per month. I didn't monitor how long the build process actually takes on average and before we run out of minutes because the workflow isn't optimised I just opted for the manual way for now.

If we have it auto build and deploy on pushes to `main` I'd also recommend to only work in side branches when new features are developed to avoid unnecessary build triggers. c:
